<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractorForm</class>
 <widget class="QWidget" name="ExtractorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="sectionLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_category">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="ck_before">
         <property name="text">
          <string>Extract Before Section</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ck_section">
         <property name="text">
          <string>Extract Section</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ck_after">
         <property name="text">
          <string>Extract After Section</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
